packages/forma-36-website/src/components/Docsearch.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect } from 'react';
+import { css } from '@emotion/core';
+import tokens from '@contentful/forma-36-tokens';
+import { TextInput, Icon } from '@contentful/forma-36-react-components';
+
+const styles = {
+  container: css`
+    position: relative;
+    display: flex;
+    align-items: center;
+    width: 100%;
+    margin-bottom: 0.5rem;
+
+    & .algolia-autocomplete {
+      width: 100%;
+    }
+  `,
+  icon: css`
+    position: absolute;
+    right: 0px;
+    display: flex;
+    margin-right: ${tokens.spacingXs};
+    z-index: 1000;
+  `,
+};
+
+const Docsearch = () => {
+  useEffect(() => {
+    if (typeof window === 'undefined') return;
+
+    window.docsearch({
```

Comment:
I think it'd be a good idea to log a message to the console if this is unavailable due to a missing API key or whatever.

Reply:
I passed a wrong API_key to see what happens and this error is printed in the console. It comes from DocSearch's script

![Screenshot 2020-04-03 at 11 16 05](https://user-images.githubusercontent.com/6597467/78345070-98acd400-759d-11ea-94a4-67a85413df2b.png)

I still added a console.error in case window.docsearch is undefined